src/pages/Systems/styles/Systems.module.css

```diff
@@ -1,253 +1,262 @@
 .sectionContainer {
-    width: 100%;
-    border: 1px solid var(--mantine-color-gray-3);
-    /* margin: 1rem; */
-    padding: 0.675rem;
-    padding-bottom: 0;
-    border-radius: 0.275rem;
+	width: 100%;
+	border: 1px solid var(--mantine-color-gray-3);
+	/* margin: 1rem; */
+	padding: 0.675rem;
+	padding-bottom: 0;
+	border-radius: 0.275rem;
 }
 
 .sectionTitleContainer {
-    flex-direction: row;
-    justify-content: space-between;
-    width: 100%;
-    padding: 1rem 1rem 0rem 0.5rem;
+	flex-direction: row;
+	justify-content: space-between;
+	width: 100%;
+	padding: 1rem 1rem 0rem 0.5rem;
 }
 
 .sectionTitle {
-    font-size: 1.2rem;
-    font-weight: 600;
+	font-size: 1.2rem;
```

Comment:
Please configure your formatting settings and let's make sure we don't commit unwanted changes.